sql-cli/sql_cli/dag_generator.py

```diff
@@ -103,15 +103,16 @@ def generate_dag(directory: Path, dags_directory: Path, generate_tasks: bool) ->
         start_date=datetime(2020, 1, 1),
         sql_files=sql_files,
     )
-    template_dag_name = "gen_tasks_dag"
     if generate_tasks:
+        template_file = Path("templates/gen_tasks_dag.py.jinja2")
+    else:
         for sql_file in sql_files_dag.sorted_sql_files():
             sql_file.write_raw_content_to_target_path()
-        template_dag_name = "render_dag"
+        template_file = Path("templates/render_dag.py.jinja2")
```

Comment:
Based on above suggestion for defining a constant for the templates directory,
```suggestion
        template_file = TEMPLATES_DIRECTORY / "render_dag.py.jinja2"
```